rust_src/src/lib.rs

```diff
@@ -36,25 +36,25 @@ extern "C" {
 }
 
 #[no_mangle]
-pub extern "C" fn rust_init_syms() {
-    unsafe {
-        defsubr(&*math::Smod);
-        defsubr(&*math::Splus);
-        defsubr(&*math::Sminus);
-        defsubr(&*math::Stimes);
-        defsubr(&*math::Squo);
-        defsubr(&*math::Slogand);
-        defsubr(&*math::Slogior);
-        defsubr(&*math::Slogxor);
-        defsubr(&*math::Smax);
-        defsubr(&*math::Smin);
-        defsubr(&*symbols::Ssymbolp);
-        defsubr(&*cons::Sconsp);
-        defsubr(&*cons::Ssetcar);
-        defsubr(&*cons::Ssetcdr);
-        defsubr(&*cons::Scar);
-        defsubr(&*strings::Sstringp);
-        defsubr(&*strings::Seq);
-        defsubr(&*strings::Snull);
-    }
+pub unsafe extern "C" fn rust_init_syms() {
```

Comment:
We really need some docs on our safety policy, I'll write something later. I think we should only have `unsafe` functions when it's possible to call them in ways that breaks memory safety. I don't think that's true here, so I propose we have an `unsafe` block inside the function.